src/Drive.php

```diff
@@ -577,6 +580,81 @@ public function emptyTrashbin(): bool
         return true;
     }
 
+    /**
+     * Invite a user or group to the drive.
+     *
+     * @param User|Group $recipient
+     * @param SharingRole $role
+     * @param \DateTimeImmutable|null $expiration
+     * @return DriveShare
+     * @throws BadRequestException
+     * @throws ForbiddenException
+     * @throws HttpException
+     * @throws InvalidResponseException
+     * @throws NotFoundException
+     * @throws UnauthorizedException
+     * @throws InternalServerErrorException
+     */
+    public function invite($recipient, SharingRole $role, ?\DateTimeImmutable $expiration = null): DriveShare
+    {
+        if((version_compare($this->ocisVersion, '6.0.0', '<'))) {
+            throw new EndPointNotImplementedException(Ocis::ENDPOINT_NOT_IMPLEMENTED_ERROR_MESSAGE);
```

Comment:
Check the "throws" in the PHPdoc above.
`EndPointNotImplementedException` needs to be added.